src/java/org/apache/ivy/plugins/report/XmlReportParser.java

```diff
@@ -71,132 +71,139 @@
 
```

Comment:
It's a change from if ... else if ... else if ... else if ... to a switch, which changes indentation and messes up git diff. I will rebase and rearrange the remaining commits.